Source/Core/DolphinQt2/Settings.cpp

```diff
@@ -1,7 +1,8 @@
-// Copyright 2015 Dolphin Emulator Project
```

Comment:
These are only to establish a date of creation. You don't need to update these when you modify them.